Descriptors/SpecialEducationSettingDescriptor.xml

```diff
@@ -90,40 +90,10 @@
 		<Description>Service provider location</Description>
 		<Namespace>uri://ed-fi.org/SpecialEducationSettingDescriptor</Namespace>
 	</SpecialEducationSettingDescriptor>
-	<SpecialEducationSettingDescriptor>
-		<CodeValue>Correctional Facilities</CodeValue>
```

Comment:
@mmarable-dlp Do you want DEPRECATED note on the other that is left behind?

Reply:
accdng to this ticket https://tracker.ed-fi.org/browse/DATASTD-1501, Correctional Facilities is one of the values for SpecialEducationSettingDescriptor